### ISSUE TITLE
EDGE release of 1.30 Issues/Suggestions

### ISSUE THREAD

**Issue by GlobalServer500** [vexorian/dizquetv#276]
I like that the 1.30 version is available now. Since it is not the stable release yet I will document some things I noticed (such as issues, errors, and suggestions).

I like how you can now use scheduled programming to automatically play the episodes/movies certain times of days and days of the week. But what I noticed is the episodes seem to go out of order or automatically go further when calculating a "time slot" schedule. This seems to be a glitch I noticed.

Error
-tv show episodes randomly just skip or go out of order after I compile all the shows and movies to create a schedule. (example season 2 episode 1 will automatically change to season 2 episode 16 after compiling the automatic schedule).

A suggestion...
-ability to add repeat to "time slot" scheduling (sometimes I like to replay an episode from earlier in the day and by default it only lets me play the next episode or shuffle the episode at a specific time).

-ability to add a channel redirect in the "time slot" scheduling as an option as well.
-deinterlace did not seem to do anything for me....not sure if this is a glitch or the video source is just unable to interlace, but I still see the interlaced episodes/video.

Hopefully these suggestions and glitches can be taken into consideration. Other than that, thanks for creating such a nice app/software for us all to use.

**Comment by vexorian** [comment]
> But what I noticed is the episodes seem to go out of order or automatically go further when calculating a "time slot" schedule

It  uses the current schedule as a base, so if the next episode in the current schedule is s04e07, it will be the first episode after applying time slots.

I don't know why they are going out of order for you.

I'd also like to have some sort of way to repeat a time slots. It's something that's planned.

You can use channel redirects in time slots, but it's looking like a lot of people haven't realized it, so I think the UX here is not intuitive enough. In order to be able to use channel redirects in time slots (or random slots) you first must add the channel redirect to the schedule before opening the slots dialog.


. Hmnn maybe some of the later updates messed up the interlace fix, can you create this report as a separate issue so that I look into it?

**Comment by vexorian** [comment]
Good point about the thursday thing. I need to make the schedule start at the current time of the day and current day.. But otherwise, yes it uses the current order that's already in the scheudle

You have to re-run the time slots after adding more shows or more episodes.

**Comment by GlobalServer500** [comment]
I will try to re-run the time slots again after adding more shows/episodes.

I will start a subchannel from scratch and try to add shows and episodes again, then create the time slots, and see if it generates from the episode position I choose as the first episode to play.
